book.bib

```diff
@@ -1413,4 +1413,14 @@ @Article{ZIBB2018
   langid = {english}
 }
 
+@article{tengeler2020gut,
```

Comment:
I suggest "Tengeler2020" as the bibtex key, to keep our reference style consistent (thus easier to remember)